src/ListMenuButton/index.js

```diff
@@ -12,6 +12,7 @@ const ListMenuButton = ({
    activeBackgroundColor,
    activeTextColor,
    activeTextWeight,
+   badgeNumber,
```

Comment:
Just curious why you chose to do this as `badgeNumber` as opposed to `rightDetail` which I think we meant to implement. And if it is separate, how should the badge number and right detail interact if used together?

A related question, should there by any styling options related to the badge? Like background color or foreground color?

Reply:
Yes, I did consider just using `rightDetail`, but for the foreseeable future, having `badgeNumber` felt like a more useful option. I can see using `rightDetail` if we created a new separate `Badge` component - but don't fully see the usefulness at the moment.

As for your second question, I had mentioned having more badge customization options in the Out of Scope above.